src/components/ShelterListItem/ShelterListItem.tsx

```diff
@@ -4,121 +4,144 @@ import { useNavigate } from 'react-router-dom';
-                :
+            <div className="flex flex-wrap gap-2" onClick={(e) => e.stopPropagation()}>
+              {Array.isArray(volunteerTags) && volunteerTags.length > 0 ? (
                 volunteerTags.map((s, idx) => (
-                  <Chip
-                    className={getSupplyPriorityProps(s.priority).className}
-                    key={idx}
-                    label={s.supply.name}
-                  />
-                ))}
+                  <WithTooltip content={nameStatusPriority(s.priority) || ''} key={idx}>
+                    <Chip
+                      className={getSupplyPriorityProps(s.priority).className}
+                      label={s.supply.name}
+                    />
+                  </WithTooltip>
+                ))
+              ) : (
+                <p>
+                  Não informado. <i> (Pode ser adicionado ao clicar no abrigo) </i>
+                </p>
+              )}
             </div>
           </div>
           <div className="flex flex-col gap-3">
             <Separator className="mt-2" />
-            <p className="text-muted-foreground text-sm md:text-lg font-medium">
+            <p className="text-sm font-medium text-muted-foreground md:text-lg">
               Necessita urgente de doações de:
             </p>
-            <div className="flex gap-2 flex-wrap">
+            <div className="flex flex-wrap gap-2" onClick={(e) => e.stopPropagation()}>
               {tags.map((s, idx) => (
-                <Chip
-                  className={getSupplyPriorityProps(s.priority).className}
-                  key={idx}
-                  label={s.supply.name}
-                />
+                <WithTooltip content={nameStatusPriority(s.priority) || ''} key={idx}>
+                  <Chip
+                    className={getSupplyPriorityProps(s.priority).className}
+                    label={s.supply.name}
+                  />
+                </WithTooltip>
               ))}
             </div>
           </div>
+
           <div className="flex flex-col gap-3">
             <Separator className="mt-2" />
-            <p className="text-muted-foreground text-sm md:text-lg font-medium">
+            <p className="text-sm font-medium text-muted-foreground md:text-lg">
               Sobrando, para doações:
             </p>
-            <div className="flex gap-2 flex-wrap">
-              {donationsTags.map((s, idx) => (
-                <Chip
-                  className={getSupplyPriorityProps(s.priority).className}
-                  key={idx}
-                  label={s.supply.name}
-                />
-              ))}
+            <div className="flex flex-wrap gap-2">
+              {Array.isArray(donationsTags) && donationsTags.length > 0 ? (
+                donationsTags.map((s, idx) => (
+                  <Chip
+                    className={getSupplyPriorityProps(s.priority).className}
+                    key={idx}
```

Comment:
Estão sendo gerados vários componentes com a mesma key, sugiro mudar par algo tipo
```suggestion
                    key={`donation-tag-${idx}`}
```